### ISSUE TITLE
Syntax Highlighting Color Contrast

### ISSUE THREAD

**Issue by shiffman** [shiffman/shiffman.net#61]
Filing issues related to accessibility. Thanks to @lunaog for reporting for the open source studio class. https://wp.nyu.edu/luna/2018/10/08/accessibility-assignment/

Syntax highlighting doesn't meet color contrast requirements for accessibility:

<img width="714" alt="screen shot 2018-10-19 at 2 22 29 pm" src="https://user-images.githubusercontent.com/191758/47236532-7e336a80-d3aa-11e8-87ed-7113ba5ac47c.png">
